crawl-ref/source/webserver/config.py

```diff
@@ -59,8 +65,20 @@
 watch_socket_dirs = False
 
 # Game configs
+#
+# You can define game configs in two ways:
+# 1. With a static dictionary `games`
+# 2. As extra games to append to this list from the `load_games` function
+#
 # %n in paths and urls is replaced by the current username
+#
 # morgue_url is for a publicly available URL to access morgue_path
+#
+# milestone_path is the path milestones are written (used for tracking active
+# game actions in the lobby list). If not set, it can be calculated from
+# dir_path. If that is also not set, there is no default. You just need to
+# specify the path to the "milestones" file, the "milestones-seeded" file next
+# to it will be automatically discovered.
 games = OrderedDict([
     ("dcss-web-trunk", dict(
```

Comment:
All existing webtiles servers have all their games config stored in this way (rather than as yaml). I figured we should show how that works rather than completely deprecating it and leaving it undocumented.